opentracing-metrics/src/main/java/io/opentracing/contrib/metrics/MetricsSpanData.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.metrics;
+
+import java.util.Map;
+
+/**
+ * This interface provides access to relevant data about a span, from which metrics can be
+ * derived.
+ *
+ */
+public interface MetricsSpanData {
```

Comment:
Sure, I have also a comment to the Interface name. It just provides access to span data no it is not associated with metrics, so I find metrics redundant there.